float32 range
uint16 servo_angle
time stamp

